artifact-manifest.md

```diff
@@ -0,0 +1,551 @@
+# OCI Artifact Manifest
+
+The OCI artifact manifest improves upon the use of OCI image-manifest, using a unique `manifest.config.mediaType`. OCI artifact manifest provides a means to define a wide range of artifacts, including a chain of dependencies of related artifacts. It provides a means to define independent artifacts, however the primary scenario is to enhance other artifact types, including enhancing OCI image-manifest and OCI image-index.
```

Comment:
Can you clarify "using a unique `manifest.config.mediaType`" here? Without context, it can be confused easily IMO

Reply:
Ahh, yes, in this context it's not clear which manifest. I'll change to `oci-image.manifest.config.mediaType`